react/ActionMenu/index.jsx

```diff
@@ -121,7 +121,8 @@ const ActionMenu = ({
       topOverlay: 'transparent',
       topBackground: theme.palette.background.paper,
       topTheme: 'dark'
-    }
+    },
+    'cozy-ui/ActionMenu'
```

Comment:
Could you split the work in 4 commits, one by feature 🙏 ?

Reply:
split into multiple commits ✅